src/webui/webapplication.cpp

```diff
@@ -200,6 +201,18 @@ WebApplication::~WebApplication()
     qDeleteAll(m_sessions);
 }
 
+#ifdef DISABLE_GUI
+void fallbackAltUI(const QString &path)
+{
+    if (path != INDEX_HTML)
+        return;
+
+    auto *preferences = Preferences::instance();
+    preferences->setAltWebUIEnabled(false);
+    preferences->apply();
+}
+#endif
+
```

Comment:
Last comment. This function should be placed in the anonymous namespace (at line 86).